WalletWasabi/Tor/Socks5/Pool/TorHttpPool.cs

```diff
@@ -331,6 +331,7 @@ private bool GetPoolConnectionNoLock(string host, ICircuit circuit, out TorTcpCo
 			// Find TCP connections to dispose.
 			foreach (TorTcpConnection tcpConnection in hostConnections.FindAll(c => c.NeedDisposal).ToList())
 			{
+				Logger.LogDebug($"['{tcpConnection}'] Is to be disposed.");
```

Comment:
Nit:

```suggestion
				Logger.LogDebug($"['{tcpConnection}'] is to be disposed.");
```

Reply:
```suggestion
				Logger.LogDebug($"['{tcpConnection}'] Connection is to be disposed.");
```

Reply:
Thanks. You are right. It should be better.